packages/node-modules-tools/src/resolve.ts

```diff
@@ -46,6 +47,15 @@ export async function resolvePackage(
     fundings = rawFunding ? [rawFunding] : []
   }
 
+  const deprecatedInfo = await getPackageDeprecatedInfo(_pkg)
```

Comment:
This is a very heavy operation and requires network access, while this function is designed to be lightweight and to work offline. We should move it to RPC, and do it on-demand like publish dates.

Reply:
Hummm, I started to look, but I'm not sure how to do it.
If it's on demand, it will not be in the graph, no ?